### ISSUE TITLE
Permanent Death

### ISSUE THREAD

**Issue by Zaxanity** [aura-project/aura#602]
On the main server, if someone dies and they don't have enough exp to revive or be revived, their exp goes into the negatives.

On Aura, if you don't have enough exp to revive in any of the three ways, you're permanently dead. This can easily be checked by setting your character level to something like 150. This stops you from reviving, even if you're a GM. I had to log out and manually set my level to be lower so that I would be able to revive.

**Comment by exectails** [comment]
Negative exp...? Are you sure about that?

**Comment by Zaxanity** [comment]
I'm very sure. If you'd like, I can rebirth on the main server so that I'm level 1 and die a bunch of times.

**Comment by xeroplz** [comment]
I can confirm this too, I remember sucking on NA and constantly being in the negative EXP zone until I pulled myself up.

**Comment by Zaxanity** [comment]
Even when you think about it, on Aura if you don't have exp to revive, there's no way for you to revive. That's a problem, isn't it?

**Comment by eotetb** [comment]
You can receive negative experience until you reach -100% experience at your level. I think at that point you can no longer revive on the spot or at the last goddess statue in a dungeon, but I am not completely sure how it works at that point.

**Comment by exectails** [comment]
@Zaxanity You are right of course, that you can't revive at all anymore is most definitely a problem. Generally you "pay" for reviving with exp, I guess I made a mistake somewhere, checking for the exp before anything else, regardless of the revive method.

I did not remember that you can have negative exp though.

**Comment by xCryptic** [comment]
I remember having negative exp a loooong time ago.

**Comment by killer000** [comment]
In aura I had negative exp but that was about -10% ~-5% or so

**Comment by Sairii** [comment]
>Negative exp...? Are you sure about that?

yes

>perma death?

OH YES PLEASE EXEC PLEASE

**Comment by killer000** [comment]
> OH YES PLEASE EXEC PLEASE

and make it possible to get perma death in PVP :P the most hardcore mabinogi server

**Comment by Sairii** [comment]
>the most hardcore mabinogi server

i have a fieldboss that has a chance to 100% kill you
